molecule/testinfra/staging/common/test_user_config.py

```diff
@@ -38,16 +40,19 @@ def test_sudoers_tmux_env(host):
     the corresponding settings there.
     """
 
-    f = host.file('/etc/profile.d/securedrop_additions.sh')
-    non_interactive_str = re.escape('[[ $- != *i* ]] && return')
-    tmux_check = re.escape('test -z "$TMUX" && (tmux attach ||'
-                           ' tmux new-session)')
+    host_file = host.file('/etc/profile.d/securedrop_additions.sh')
+    source_file = os.path.abspath(
+        os.path.join(
+            os.path.dirname(__file__),
+            '../../../../install_files/securedrop-config/etc/profile.d',
+            'securedrop_additions.sh'
+        )
+    )
+    expected_content = open(source_file).read()
+    h = hashlib.sha256()
+    h.update(expected_content)
 
-    assert f.contains("^{}$".format(non_interactive_str))
-    assert f.contains("^if which tmux >\/dev\/null 2>&1; then$")
-
-    assert 'test -z "$TMUX" && (tmux attach || tmux new-session)' in f.content
-    assert f.contains(tmux_check)
+    assert host_file.sha256sum == h.hexdigest()
```

Comment:
Your motivation to DRY out the config tests here is understandable, but most of the config tests are _intentionally_ duplicative, to enable us to refactor the provision logic confidently. For example, if you modify `install_files/securedrop-config/etc/profile.d/securedrop_additions.sh` locally, then build packages and install them, you should reasonably expect the corresponding config tests to fail. These changes would cause them to pass. 

A broader discussion is warranted here about the pattern used for hardcoding values in the config tests. For now, simply for the sake of consistency, I'd prefer to keep them. Even a multiline string asserting an exact match on `host_file.contents` or `host_file.content_string` would be sufficient.

Reply:
Reverted to comparing the exact current content.